curp/src/server/raw_curp/mod.rs

```diff
@@ -618,6 +641,8 @@ impl<C: 'static + Command, RC: RoleChange + 'static> RawCurp<C, RC> {
 
         let prev_last_log_index = log_w.last_log_index();
         self.recover_from_spec_pools(&mut st_w, &mut log_w, spec_pools);
+        let propose_id = ProposeId(rand::random(), 0);
```

Comment:
Should this id generation use the same way of the client id refactor(client id need to be acquired from server). If so please add a todo.

Reply:
Done